docs/python-grader/Makefile

```diff
@@ -0,0 +1,9 @@
+SPHINXOPTS    ?= -c sphinx-config
+SPHINXBUILD   ?= sphinx-build
```

Comment:
Is this something that should be included into the `centos7-plbase` image?

Reply:
I've just been running it manually, but having it auto build in docker is probably a better idea

Reply:
I'm happy to run it manually (probably the best option at the moment until we set up a proper docs build system) but it would be nice if we can run it inside the PL docker container without installing extra things.

Reply:
That is, `cd /PrairieLearn/docs ; make` should "just work" inside the PL docker container.